esmvaltool/cmorizers/data/formatters/datasets/ncep_ncar_r1.py

```diff
@@ -143,6 +143,9 @@ def _extract_variable(short_name, var, cfg, raw_filepath, out_dir):
 
     cube = _fix_coordinates(cube, definition, cmor_info)
 
+    if var.get("make_negative"):
+        cube = -1 * cube
```

Comment:
this seemed to make the file lose some attributes - when loading the new files had warnings and esmvalcore generic fixes added standard name and long name. you could try accessing `cube.data = -1 * cube.data` instead or move before all the attributes are set

Reply:
Fixed